cmd/ingest.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"go/types"
+
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/network"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/ingest"
+)
+
+type ingestCmd struct{}
+
+func (c *ingestCmd) Command() *cobra.Command {
+	cfg := ingest.Configs{}
+	cfgOpts := config.ConfigOptions{
+		{
+			Name:        "database-url",
+			Usage:       "Database connection URL.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.DatabaseURL,
+			FlagDefault: "postgres://postgres@localhost:5432/wallet-backend?sslmode=disable",
+			Required:    true,
+		},
+		{
+			Name:        "network-passphrase",
+			Usage:       "Stellar Network Passphrase to connect.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.NetworkPassphrase,
+			FlagDefault: network.TestNetworkPassphrase,
+			Required:    true,
+		},
+		{
+			Name:        "captive-core-bin-path",
+			Usage:       "Path to Captive Core's binary file.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.CaptiveCoreBinPath,
```

Comment:
I was thinking about adding a `CustomSetValue` for it since it's a folder, we could add validations to see if the folder exists before the application starts.

Reply:
Good one! [c81a985](https://github.com/stellar/wallet-backend/pull/14/commits/c81a98518778990bedbd71a3f5afa01968b66715)